burr/integrations/serde/pandas.py

```diff
@@ -0,0 +1,31 @@
+# try to import to serialize Pandas Objects
+import hashlib
+import os
+
+import pandas as pd
```

Comment:
Good to use the import guards we have above. We can add it for debugging and catch it more explicitly. Not essential though.

Reply:
We don't want them here as they aren't user imported. The framework tries to import them. So the message is moot.

Reply:
Sure, just figured it would be nice for consistency, but you're right it is not the same.